week-1-practise/level-1/9-function-output-2.js

```diff
@@ -6,9 +6,22 @@ var result = s('code', 'your');
 console.log(result);
 
 // Answer these questions:
-// - What does this program do? Run it and see, or Google some of the methods used ot understand them.
+// - What does this program do? Run it and see, or Google some of the methods used ot understand them. 
+//it will join strings and give back a new string with the sum result. In arrays, it will join arrays by order
+//and create new array with the elements of each parameter 
 // - How many parameters does the function take?
+//as many as wanted
 // - What is the function name?
+//s
 // - Where is the function called? with what parameters?
+//line 6, code, your
 // - What does the function return?
-// - How can this function be improved?
\ No newline at end of file
+//a new string with a joined strings codeyour
+// - How can this function be improved?
+//by giving a more clear names for the functions and the paramters as well as adding a space between both strings
```

Comment:
I definitely agree on the name! What would you call it?

Without knowing what the function is going to be used for, it's hard to say whether the space is better. It does seem to make more sense for the example given doesn't it.